wdfn-server/waterdata/tests/test_camera.py

```diff
@@ -4,7 +4,9 @@
 from unittest import TestCase
 from .. import camera
 
-camera.JSON_CAMERA_FILE = "wdfn-server/data/monitoring_camera_data_test.json"
+JSON_CAMERA_TEST_FILE = "wdfn-server/data/monitoring_camera_data_test.json"
+with open(JSON_CAMERA_TEST_FILE, 'r') as json_file:
+     camera.MONITORING_CAMERA_RAW = json_file.read()
```

Comment:
Not a huge deal, but I'm wondering if it would be cleaner to mock out this constant rather than replacing it.

Reply:
I am all for it. Anything in particular you had in mind?